lib/processor.js

```diff
@@ -32,6 +32,12 @@
 			var pxre = /([\d\.]+)\D*/;
 			var width = svgelem.getAttribute( "width" );
 			var height = svgelem.getAttribute( "height" );
+			var viewbox = svgelem.getAttribute( 'viewBox' );
+			if (viewbox && width === '100%' && height === '100%') {
```

Comment:
What about the case when there is a viewbox and no width/height added? I think defaulting to `100%` might be smart there (meaning, I'm ok with dumping default sizes in the case of a viewbox being included - I'm pretty sure that's expected behavior for an svg at least)

Maybe that's worth a major release later (as it will break some functionality)? cc @johnbender, what do you think?